coral/.husky/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+STAGED_FILES=$(git diff --staged --name-only)
+
+if echo "$STAGED_FILES" | grep -q coral;
+then
+  echo 'Files on coral staged'
+    cd coral || return
```

Comment:
Instead of changing the directory, I think we could try to following:
```
pnpm run --prefix coral lint-staged
pnpm run --prefix coral test --bail
```